packages/react-components/src/templates/TutorialDetailsPage.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ComponentProps } from 'react';
+import { css } from '@emotion/react';
+import { TutorialsResponse } from '@asap-hub/model';
+
+import { Card, Headline2, Divider, Paragraph } from '../atoms';
+import { perRem } from '../pixels';
+import { contentSidePaddingWithNavigation } from '../layout';
+import { TagList } from '../molecules';
+import {
+  RelatedTutorialsCard,
+  RichText,
+  TutorialAdditionalInformationCard,
+  TutorialHeaderCard,
+  RelatedEventsCard,
+  HelpSection,
+} from '../organisms';
+
+const containerStyles = css({
+  padding: `${36 / perRem}em ${contentSidePaddingWithNavigation(8)}`,
+});
+
+const tagsContainer = css({
+  margin: `${12 / perRem}em 0 ${12 / perRem}em 0`,
+});
+
+const cardsStyles = css({
+  display: 'grid',
+  rowGap: `${36 / perRem}em`,
+});
+
+type TutorialDetailsPageProps = Pick<
+  TutorialsResponse,
+  'text' | 'tags' | 'relatedTutorials' | 'relatedEvents'
+> &
+  ComponentProps<typeof TutorialHeaderCard> &
+  ComponentProps<typeof TutorialAdditionalInformationCard>;
+
+const TutorialDetailsPage: React.FC<TutorialDetailsPageProps> = ({
+  text,
+  relatedTutorials,
+  relatedEvents,
+  tags,
+  ...props
+}) => {
+  const hasAdditionalInformation = [
+    props.asapFunded,
+    props.sharingStatus,
+    props.usedInPublication,
+  ].some((info) => info != null);
+
+  return (
+    <div css={containerStyles}>
+      <div css={cardsStyles}>
+        <TutorialHeaderCard {...props} />
+        {(text || !!tags.length) && (
```

Comment:
Why are description and tags dependent on each other? Should it not be we show description if it exist regardless if tags exists and vice versa.

Reply:
We don't want to display the card at all if neither is present